### PR TITLE
chore(agw): s1ap_utils.py is cleaned up

### PR DESCRIPTION
**Pull request by mpfirrmann**
## Summary

This is a clean-up PR for the `s1ap_utils.py`. After extensive work in #13853, we shuffled some functions around for better visibility and addressed several `pylint` remarks.

## Test Plan

- The LTE integration tests were run on my fork with GitHub actions with a `systemd`-base AGW; [LINK](https://github.com/mpfirrmann/magma/actions/runs/3081265323).

- Tests ran green for the containerized AGW as well.

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Review by voisey — COMMENTED** [review]
How was pylint run on this PR? Was it according to some configuration in magma or using default settings?

**Comment by mpfirrmann** [comment]
> How was pylint run on this PR? Was it according to some configuration in magma or using default settings?

Default settings. The main motivation for this PR was the reordering of the functions and after that it was more a Boy's Scout thing to clean up. We just wanted to separate these changes from the feature PR.

**Comment by voisey** [comment]
@mpfirrmann Have you checked the S1AP tests still run after these changes?

**Comment by mpfirrmann** [comment]
> @mpfirrmann Have you checked the S1AP tests still run after these changes?

Yes. I edited the PR description

**Review by voisey — APPROVED** [review]

**Comment by mpfirrmann** [comment]
@rsarwad, do you maybe have some time to take a look at this PR?

File: lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py [hunk 1]
```diff
@@ -590,7 +594,12 @@ def _verify_dl_flow(self, dl_flow_rules=None, ipv6_non_nat=False):
                         )
                         assert bool(has_tunnel_action)
 
-    def verify_flow_rules(self, num_ul_flows, dl_flow_rules=None, ipv6_non_nat=False):
+    def verify_flow_rules(
+        self,
+        num_ul_flows,
+        dl_flow_rules=None,
+        ipv6_non_nat=False,
+    ):
```
Comment (rsarwad) [inline lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py:604]: At line 626, can we use curly brackets to get the values?

**Review by rsarwad — APPROVED** [review]
Had minor query. Rest of the changes look good to me

**Pull request merged**